www/_template/guides/plugins.md

```diff
@@ -246,3 +269,4 @@ In general, make sure to mind the following checklist:
 - If `load()` doesn't return anything, the file isn’t loaded and the `load()` of the next suitable plugin is called.
 - If `transform()` doesn't return anything, the file isn’t transformed.
 - If you want to build a plugin that runs some code only on initialization (such as `@snowpack/plugin-dotenv`), put your side-effect code inside the function that returns your plugin. But be sure to still return a plugin object. A simple `{ name }` object will do.
```

Comment:
why is everything red :(

Reply:
It looks like it‘s marking the `- ` as a diff (removal) for some reason

Reply:
oh wait IDK. that‘s weird. It‘s almost like there‘s a syntax error in that json code block, and it‘s trying to color this as JSON. But I don‘t see any syntax errors